arviz/stats/stats.py

```diff
@@ -792,6 +804,18 @@ def r2_score(y_true, y_pred):
     Pandas Series with the following indices:
     r2: Bayesian R²
     r2_std: standard deviation of the Bayesian R².
+
+    Examples
+    --------
+    Calculate R² for Bayesian regression models :
+
+    .. ipython::
+
+        In [1]: import arviz as az
+           ...: y_true=np.random.randn(10)
```

Comment:
maybe we can use this example

data = az.load_arviz_data('regression1d')
y_true = data.observed_data["y"].values
y_pred = data.posterior_predictive.stack(sample=("chain", "draw"))['y'].values.T
az.r2_score(y_true, y_pred)

Reply:
Yes sure. I will make changes.